### PR TITLE
docs: add openapi spec docs in examples and improve the docs in services

### PR DESCRIPTION
**Pull request by sumiter92**
## Description

Add openapi spec docs in examples and improve the documentation in services and examples.

Fixes # 211

## Type of change

Please delete options that are not relevant.

- [ ] Intermediate change (work in progress)

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated

### CODE REVIEW

**Comment by samarpan-b** [comment]
Not following conventional commit guidelines

**Comment by samarpan-b** [comment]
https://github.com/sourcefuse/loopback4-microservice-catalog/blob/master/DEVELOPING.md#commit-message-guidelines

**Comment by samarpan-b** [comment]
We also need to replace mention of loopback4-microservices-catalog with Sourceloop, all through the readmes. Also add link to https://github.com/sourcefuse/loopback4-microservice-catalog/blob/master/DEVELOPING.md in root readme

File: services/audit-service/README.md [hunk 1]
```diff
@@ -166,3 +166,5 @@ Authorization: Bearer <token> where <token> is a JWT token signed using JWT issu
 
 #### API Details
 
+Visit the [OpenAPI spec docs](../../sandbox/audit-ms-example/openapi.md)
```
Comment (samarpan-b) [inline services/audit-service/README.md:169]: why audit example ?

File: services/bpmn-service/README.md [hunk 2]
```diff
@@ -227,3 +227,7 @@ Endpoint to delete a workflow, uses the `delete` method from the provider.
 
 `GET /workflow`
 Endpoint to get all the workflows.
+
+## API's Details
+
+Visit the [OpenAPI spec docs](../../sandbox/workflow-ms-example/openapi.md)
```
Comment (samarpan-b) [inline services/bpmn-service/README.md:233]: why ref to example ?

**Review by samarpan-b — CHANGES_REQUESTED** [review]

File: services/bpmn-service/README.md (hunk 2 above)
Comment (sumiter92) [reply services/bpmn-service/README.md]: modified

File: services/audit-service/README.md (hunk 1 above)
Comment (sumiter92) [reply services/audit-service/README.md]: modified

**Comment by sumiter92** [comment]
> We also need to replace mention of loopback4-microservices-catalog with Sourceloop, all through the readmes. Also add link to https://github.com/sourcefuse/loopback4-microservice-catalog/blob/master/DEVELOPING.md in root readme

done

**Pull request merged**